### PR TITLE
Update changelog and cli version

### PR DESCRIPTION
**Pull request by calcaide**
## Description

Update changelog and cli version.

### CODE REVIEW

File: ui/desktop/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
+## V2.0.3 [CLI 0.16.0] (2024.03.30)
+
+### New
+
+- Upgrade CLI to v0.16.0 ( [PR](https://github.com/hashicorp/boundary-ui/pull/2280) )
```
Comment (ZedLi) [inline ui/desktop/CHANGELOG.md:9]:
```suggestion
- Upgrade CLI to v0.16.0 ([PR](https://github.com/hashicorp/boundary-ui/pull/2280))
```

I think these parentheses have extra white space

File: ui/desktop/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+- Add Aliases support ( [PR](https://github.com/hashicorp/boundary-ui/pull/2256) )
```
Comment (ZedLi) [inline ui/desktop/CHANGELOG.md:10]:
```suggestion
- Add Aliases support ([PR](https://github.com/hashicorp/boundary-ui/pull/2256))
```

File: ui/desktop/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
+- ARM64 Support.
```
Comment (ZedLi) [inline ui/desktop/CHANGELOG.md:11]: Should we specify for mac/darwin?

File: ui/desktop/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,18 @@
...
+
+### Bug Fixes
+
+- Fix client daemon search for Windows ( [PR](https://github.com/hashicorp/boundary-ui/pull/2245) )
```
Comment (ZedLi) [inline ui/desktop/CHANGELOG.md:15]:
```suggestion
- Fix client daemon search for Windows ([PR](https://github.com/hashicorp/boundary-ui/pull/2245))
```

**Review by ZedLi — APPROVED** [review]

File: ui/desktop/CHANGELOG.md (hunk 3 above)
Comment (cameronperera) [reply ui/desktop/CHANGELOG.md:11]: I would to be safe.

**Review by cameronperera — APPROVED** [review]

File: ui/desktop/CHANGELOG.md (hunk 3 above)
Comment (calcaide) [reply ui/desktop/CHANGELOG.md:11]: 😅 sorry guys, you totally right.

**Review by lisbet-alvarez — APPROVED** [review]
🥳🎊

**Review by DhariniJeeva — APPROVED** [review]

**Pull request merged**